src/Apple/Provider.php

```diff
@@ -195,8 +195,10 @@ public function user()
      */
     protected function mapUserToObject(array $user)
     {
-        if (request()->filled('user')) {
-            $userRequest = json_decode(request('user'), true);
+        $value = trim((string) request('user'));
```

Comment:
Could we do an early return here?

```php
if(empty(request('user')) {
    return null
}
```

Reply:
we can't because it would change the signature of the method, it should always return an instance of `Laravel\Socialite\Two\User`, as defined in `Laravel\Socialite\Two\AbstractProvider` where this method is inherited from

P.S.: in this case it returns an instance of `SocialiteProviders\Manager\OAuth2\User` which extends from `Laravel\Socialite\Two\User`